srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -96,9 +101,62 @@ public Expression getParsedExpression()
         return fBooleanExpression;
     }
     
+    /**
+     * Returns true if this matcher should be evaluated and false if it should not
+     * be evaluated.
+     */
+    @Basic
+    public boolean isEnabled()
+    {
+        return fEnabled;
+    }
+    
+    void setEnabled(final boolean enabled)
+    {
+        fEnabled = enabled;
+    }
+    
+    /**
+     * Evaluate this ValueMatcher's boolean expression based on the given context and value.
+     * @param context the EvaluationContext used to evaluate this matcher against
+     * @param value the value for this matcher's variable
+     * @return the boolean value to which the expression evaluates
+     */
     public boolean evaluate(final EvaluationContext context, final Value value)
     {
-        return fBooleanExpression.getValue(context, value, Boolean.class);
+        if(fEnabled && !fBooleanExpression.getExpressionString().isEmpty())
```

Comment:
Because it is valid to have a blank expression. If the expression is blank, it defaults to true. For instance, if the apply condition flag is false then there doesn't need to be any expression present.